### ISSUE TITLE
Consider adding shortcut for s(by.xpath("")) as sx("")

### ISSUE THREAD

**Issue by SergeyPirogov** [yashaka/selene#110]
In chrome dev tools console it's possible to query elements using **$("css selector")**, also 
it's possible to use xpath as **$(x("xpath selector"))**.

Maybe we should add alias to be able to use **s(x("xpath locator"))** or **sx("xpath locator")**.

It will be much easier to write tests:

- s(by.xpath("//a"))

- sx("//a")

P/S Initial idea comes from meetup where I showed Selene to other people

**Comment by yashaka** [comment]
Selenide added this in latest version:

=== 4.4 (released 27.03.2017)
* #497 Added methods `$x("//div")`, `$$x("//div")` for finding elements by xpath 

But I am still unsure if the following code would be not too much:)

```python
s('#new-item'). ...
ss('.item'). ...
sx('//div'). ...
ssx('//*[class="special"]'). ...
```

in python it does not look same way readable as in Java and JS with `$`

But to make the following syntax seems not so bad:

```python
s('#new-item'). ...
ss('.item'). ...
s(x('//div')). ...
ss(x('//*[class="special"]')). ...
```

Though I would still not include default import of `x` in the `selene.api`, in order to encourage people not to use xpath. Selene has all alternatives to xpath that are much more readable...

**Comment by SergeyPirogov** [comment]
Yes, I agree. $x() was my proposal to Selenide =) In Python it's easier to add custom method's like *s(x())* or *s(by.model())*, so experienced people can use such approach. Let's have this for history, but not inclide in main selene api
